### PR TITLE
Update browserContext.waitForEvent

### PR DESCRIPTION
**Pull request by ankur22**
Updates the `browserContext.waitForEvent` docs to reflect the changes in https://github.com/grafana/xk6-browser/pull/1042 and https://github.com/grafana/xk6-browser/pull/1069.

### CODE REVIEW

**Review by inancgumus — APPROVED** [review]
Nice, thanks.

**Comment by ka3de** [comment]
I think this branch should target `release-notes-v0.48.0` branch.

File: src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/02 BrowserContext/waitForEvent.md [hunk 1]
```diff
@@ -1,33 +1,71 @@
...
-</Blockquote>
+</TableWithNestedRows>
 
-Waits for the event to fire and passes its value into the predicate function. Returns the event data value when the predicate returns `true`.
+### Returns
 
-<TableWithNestedRows>
+| Type            | Description                                                                                           |
+|-----------------|-------------------------------------------------------------------------------------------------------|
+| Promise<Object> | At the moment a [Page](/javascript-api/k6-experimental/browser/page/) object is the only return value |
```
Comment (ka3de) [inline src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/02 BrowserContext/waitForEvent.md:23]: I believe we have to escape these `<` `>`. See related CI error [here](https://github.com/grafana/k6-docs/actions/runs/6669691353/job/18128069484?pr=1384#step:10:49).

Comment (ankur22) [reply src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/02 BrowserContext/waitForEvent.md]: Thanks for that, fixed in a fixup e49bca0afaca619b1466fa59b84abd28bf800361

**Review by ka3de — APPROVED** [review]
LGTM.

**Comment by inancgumus** [comment]
The branch should be named `release-v0.48.0`, not `release-notes-0.48.0`. FYI: @grafana/k6-browser

**Pull request merged**